Comment out svc-bie-kafka workflows tied to inactive services

gabezurita opened the pull request:
- Prevent unnecessary execution and dependency/security updates.
- Added comments with context and links to relevant documentation:
  - Disabling issue: #3802
  - BIE Docs: #3753

gabezurita committed:
Comment out svc-bie-kafka workflows tied to inactive services

- Prevent unnecessary execution and dependency/security updates.
- Added comments with context and links to relevant documentation:
  - Disabling issue: #3802
  - BIE Docs: #3753

brostk commented on docker-compose.yml:
```diff
@@ -158,25 +158,29 @@ services:
+  #   depends_on:
+  #     <<: [ *svc-depends-on ]
 
   vro-streamlit:
```
We could probably comment out streamlit things too, any thoughts @dfitchett?

gabezurita replied:
We might as well. I'll push up a commit after my latest commit goes fully green 😄

dfitchett replied:
Yeah, but i think that can probably happen separately! Lets keep our scope narrowed to this service.

gabezurita replied:
Whoops, I pushed up a commit doing it:
https://github.com/department-of-veterans-affairs/abd-vro/pull/3814/commits/97a81b399365a6c4f83db0dcb23a6e5f3d6c1ae1

I'm happy to split it off or merge it.

dfitchett replied:
Nah no worries!

brostk commented on .github/workflows/container-healthchecks.yml:
```diff
@@ -39,14 +39,18 @@ jobs:
+      #     # Build and start Kafka mock required for svc-bie-kafka
+      #     ./gradlew -p mocks :mock-bie-kafka:docker
+      #     COMPOSE_PROFILES="kafka" ./gradlew -p mocks :dockerComposeUp
+      #     COMPOSE_PROFILES="platform,svc" ./gradlew :dockerComposeUp
```
We should still use COMPOSE_PROFILES="platform,svc" for the next builds, it looks like ep-merge is [failing](https://github.com/department-of-veterans-affairs/abd-vro/actions/runs/12130970180/job/33822290526?pr=3814#step:10:41) its health check further down in this action.  I imagine it's because our other platform services aren't running.

gabezurita replied:
A yes, let's see if [this commit](https://github.com/department-of-veterans-affairs/abd-vro/pull/3814/commits/64c2f4d26db9788de4ee598985c22f6d234011cc) fixes the failure.

Yep, that did the job: https://github.com/department-of-veterans-affairs/abd-vro/actions/runs/12143015588/job/33858886640?pr=3814

gabezurita committed:
Fix .github/workflows/container-healthchecks
- Added explicit execution of `COMPOSE_PROFILES="platform,svc" ./gradlew :dockerComposeUp` to ensure necessary platform and svc services are started.

Relevant error:
vro_ee-ep-merge-app_1 Up About a minute (unhealthy) There are unexpected unhealthy containers! Error: Process completed with exit code 2.

See PR comment: https://github.com/department-of-veterans-affairs/abd-vro/pull/3814#discussion_r1867833913

gabezurita committed:
disable vro-streamlit in workflows
https://github.com/department-of-veterans-affairs/abd-vro/pull/3814#discussion_r1867826060

dfitchett reviewed (approved):
LGTM
